test_human.py

```diff
@@ -0,0 +1,30 @@
+from crawler.rss import RSSCrawler
```

Comment:
Add comment at the top of the file saying

"This test is for developers to run manually"

Reply:
OK. It'll be fixed on next PR.